model/stats/db.go

```diff
@@ -442,15 +443,350 @@ func statsToUpdatePipeline(projectId string, start time.Time, end time.Time) []b
 			"day":        bson.M{"$dateFromString": bson.M{"dateString": "$_id.day", "format": "%Y-%m-%d"}},
 			"task_names": 1,
 		}},
-		{"$sort": bson.M{
-			"project":   1,
-			"date":      1,
-			"requester": 1,
+		{"$sort": bson.D{
+			{Name: "project", Value: 1},
```

Comment:
These should use `ProjectKey`, etc., as should the rest of this PR.

Reply:
Done.